packages/components/src/components/languageSupport/lezerDebug.ts

```diff
@@ -0,0 +1,105 @@
+import { Input, NodeType, Tree, TreeCursor } from "@lezer/common";
+
+enum Color {
+  Red = 31,
+  Green = 32,
+  Yellow = 33,
+}
+
+function colorize(value: any, color: number): string {
```

Comment:
What's the "any" here? Why does it return only a String of the value? This method seems unnecessary / a development artifact.

Reply:
Yeah, it's been something that did colored strings in a terminal, and is not used anymore. That whole file is for debugging the parser, and needs to be moved somewhere afterwards